code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -506,6 +505,93 @@
 		new /obj/item/stack/sheet/metal(loc, 4)
 	qdel(src)
 
+/**
+  * # Food and Drink Cart
+  *
+  * Secure variant of the [Food Cart][/obj/machinery/smartfridge/foodcart].
+  */
+/obj/machinery/smartfridge/secure/foodcart
+	name = "E.A.T."
+	desc = "Short for the Entrée Armoured Transport, this technological marvel by Donk Co. allows Chefs to hawk their food and drink wares around the station, just with a bit more security. Swipe an authorised ID to engage the bluespace-plastitanium anchors."
```

Comment:
```suggestion
	desc = "Short for the Entrée Armored Transport, this technological marvel by Donk Co. allows Chefs to hawk their food and drink wares around the station, just with a bit more security. Swipe an authorized ID to engage the bluespace-plastitanium anchors."
```
Moving stuff to American English, as it is more consistent with the rest of the code base.